fakefs.gemspec

```diff
@@ -9,14 +6,11 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Chris Wanstrath", "Scott Taylor", "Jeff Hodges", "Pat Nakajima", "Brian Donovan"]
   spec.email         = ["chris@ozmm.org"]
   spec.description   = %q{A fake filesystem. Use it in your tests.}
-  spec.summary       = %q{A fake filesystem. Use it in your tests.}
-  spec.homepage      = "http://github.com/defunkt/fakefs"
+  spec.summary       = spec.description
+  spec.homepage      = "https://github.com/defunkt/fakefs"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files`.split($/)
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-  spec.require_paths = ["lib"]
+  spec.files         = `git ls-files lib README.markdown LICENSE`.split($/)
```

Comment:
Generally it is a good idea to also ship tests with a gem. Package maintainers of Linux distributions usually run tests as a part of the distro package building process.

See the [Debian Ruby packaging team's notes on testing](https://wiki.debian.org/Teams/Ruby/Packaging/Tests#Case_four:_missing_test_folder_from_rubygems_org) for an example.